[Bug] OpenVINO master failed to compile on x64 Windows using Visual Studio 2017 toolchain

##### System information (version)

- OpenVINO => [d681582](https://github.com/openvinotoolkit/openvino/commit/d681582251f8703d0e1f403258e61e2e2bbd91f7)
- Operating System / Platform => Windows 10 64 Bit
- Compiler => Visual Studio 2017
- Problem classification: Compilation from sources

##### Detailed description

OpenVINO compilation fails with the following error:
```
C:\repo\openvino-master\openvino\ngraph\core\src\op\util\attr_types.cpp(169): error C2039: 'tolower': is not a member of 'std' [C:\repo\openvi
no-master\openvino\build\ngraph\core\ngraph.vcxproj]
```

##### Steps to reproduce

Follow [building for Windows guide](https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows) on Windows 10 64-bit host with Visual Studio 2017.

--- a/ngraph/core/src/op/util/attr_types.cpp
+++ b/ngraph/core/src/op/util/attr_types.cpp
@@ -14,6 +14,7 @@
 // limitations under the License.
 //*****************************************************************************
 #include <map>
+#include <cctype>
 
 #include "ngraph/attribute_visitor.hpp"
 #include "ngraph/check.hpp"

--- a/ngraph/core/src/op/util/attr_types.cpp
+++ b/ngraph/core/src/op/util/attr_types.cpp
@@ -13,8 +13,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //*****************************************************************************
-#include <map>
 #include <cctype>
+#include <map>
 
 #include "ngraph/attribute_visitor.hpp"
 #include "ngraph/check.hpp"

